### DISCUSSION TITLE
モバイル学習コンテンツで、演習課題の実装内容を確認したい

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#263] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/screens/exercise
演習課題

演習課題の実装内容を確認したい。
（実装が正しいか確認したい。存在するのであればリンクがほしい）

**Comment by nagano-ryo** [comment]
以下の検討事項あり。
- 完成形があったとしてdiffが取りづらい。（演習課題ごとに必要になってしまう。）
- ヒントがあっても良いかも。
- 「わからなかったらこのページを復習しましょう」があっても良い。
- そもそも過去に演習課題で詰まった人はいないと思われる。

**Comment by creature-water-valley** [comment]
現状は、受講者から依頼を受けてレビューする運用としている。
この運用で問題があればまだ新規Discussionを上げて考える。
